README.md

```diff
@@ -19,14 +19,21 @@ Replace the placeholder with your Android SDK location, ex: `/User/{USER_NAME}/L
 
 [iOS Specific](https://github.com/realeyes-media/scte35-swift)
 
+[Javascript Specific](https://github.com/realeyes-media/scte35-js)
+
+
+
+### Test Harnesses
+
 [iOS Test Harness](https://github.com/realeyes-media/scte35-decoder-multiplatform-iOS-harness)
```

Comment:
The other "Android specific" and "JS specific" links are to test harnesses so this link can actually be grouped with them under the "Reference Projects" section